ocaml/idl/datamodel_host.ml

```diff
@@ -878,7 +878,7 @@ let host_query_ha = call ~flags:[`Session]
       ~pool_internal:true
       ~hide_from_docs:true
       ~name:"certificate_install"
-      ~doc:"Install an SSL certificate to this host."
+      ~doc:"Install a TLS CA certificate to this host."
```

Comment:
While we are here; "install .. to" seems to me unusual. A native speaker should comment on this but "install on", "install for" seem to me conventional.